haproxy_exporter.go

```diff
@@ -211,6 +213,10 @@ func NewExporter(uri string, selectedServerMetrics map[int]*prometheus.GaugeVec,
 			42: newBackendMetric("http_responses_total", "Total of HTTP responses.", prometheus.Labels{"code": "4xx"}),
 			43: newBackendMetric("http_responses_total", "Total of HTTP responses.", prometheus.Labels{"code": "5xx"}),
 			44: newBackendMetric("http_responses_total", "Total of HTTP responses.", prometheus.Labels{"code": "other"}),
+			58: newBackendMetric("http_queue_time", "Avg. HTTP queue time for last 1024 successful connections.", nil),
```

Comment:
Yeah, maybe I'd put the average in the end though, how about: `http_queue_time_average_seconds` and similar for the other ones.